src/WaterPokemon.java

```diff
@@ -0,0 +1,82 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class WaterPokemon extends Pokemon {
+
+    private static final List<String> attacks = Arrays.asList("surf", "hydroPump", "hydroCanon", "rainDance");
+
+    private static final String type = "water";
+
+    public WaterPokemon(String name, int level, int hp, String food, String sound) {
+        super(name, level, hp, food, sound, type);
+    }
+
+    public List<String> getAttacks() {
+        return attacks;
+    }
+
+    void surf(Pokemon name, Pokemon enemy) {
```

Comment:
het is netter om hier public voor te zetten. zo is het duidelijk voor de programmeur waar deze functie gebruikt kan worden.